.gitignore

```diff
@@ -1,2 +1,4 @@
 *.html
 *.txt
+
+*~
```

Comment:
Sorry this should may not have been in there. This is to filter out all the temporary files that vim creates when you edit a doc.